beetsplug/discogs.py

```diff
@@ -315,30 +339,95 @@ def get_tracks(self, tracklist):
 
         return tracks
 
+    def coalesce_tracks(self, raw_tracklist):
```

Comment:
Oh wow, that's pretty crazy! Perhaps we should flatten these subtrack lists in a subsequent round of changes.